### ISSUE TITLE
[Feature Request] Tabs with dropdowns

### ISSUE THREAD

**Issue by hlKittipan** [ElemeFE/element#21836]
### Existing Component
Yes

### Component Name
Tabs

### Description
should have Tabs with dropdowns
 https://getbootstrap.com/docs/5.1/components/navs-tabs/#tabs-with-dropdowns

**Comment by LSL1618** [comment]
like this :
```
<el-tabs type="border-card">
    <el-tab-pane>
        <span slot="label">
            <el-dropdown>
                <el-button type="primary" size="small" plain>
                    Country<i class="el-icon-arrow-down el-icon--right"></i>
                </el-button>
                <el-dropdown-menu slot="dropdown">
                <el-dropdown-item>Province</el-dropdown-item>
                <el-dropdown-item>City</el-dropdown-item>
                <el-dropdown-item>District</el-dropdown-item>
                </el-dropdown-menu>
            </el-dropdown>
        </span>
    </el-tab-pane>
</el-tabs>
```

**Comment by hlKittipan** [comment]
> like this :
> 
> ```
> <el-tabs type="border-card">
>     <el-tab-pane>
>         <span slot="label">
>             <el-dropdown>
>                 <el-button type="primary" size="small" plain>
>                     Country<i class="el-icon-arrow-down el-icon--right"></i>
>                 </el-button>
>                 <el-dropdown-menu slot="dropdown">
>                 <el-dropdown-item>Province</el-dropdown-item>
>                 <el-dropdown-item>City</el-dropdown-item>
>                 <el-dropdown-item>District</el-dropdown-item>
>                 </el-dropdown-menu>
>             </el-dropdown>
>         </span>
>     </el-tab-pane>
> </el-tabs>
> ```

Yes some think like that. but when selected dropdown menu should be tab.
